server/bin/unittests

```diff
@@ -224,7 +224,7 @@ function _dump_logs {
     # Be quiet if there's no database, or if the `dataset` table hasn't been
     # created.
     db=${_testroot}/test_db
-    select='select (select users.username from users where users.id = datasets.owner_id),controller,name,state,(select " " || key || " = " || value from dataset_metadata where dataset_ref = datasets.id) from datasets order by (select users.username from users where users.id = datasets.owner_id),controller,name asc'
+    select='select (select users.username from users where users.id = datasets.owner_id),controller,name,state,(select " " || key || " = " || left(value,80) from dataset_metadata where dataset_ref = datasets.id) from datasets order by (select users.username from users where users.id = datasets.owner_id),controller,name asc'
```

Comment:
I'm conflicted on truncating the `value` column. The enormous output from index maps is really hard to read, so truncating it makes reviewing the diffs a lot easier. On the other hand, it also means we're not validating the entire output, and once the gold files are updated we generally won't have to see it at all.